src/assets/docs/faq.md

```diff
@@ -112,6 +112,22 @@ We also strive to use this to highlight tools that share a common set of recomme
 
 This is going to be a moving quality target as tools improve, but you have a tool that you wish to be verified, please send us a heads-up via our GitHub issues or Gitter!
 
+## What is a default version of a tool or workflow?
+
+Every tool/workflow is recommended to have a default version set by its author.  It indicates to the users which version of the tool/workflow they should use.
+For tools, the default version is uniquely identified by the tag from the Docker image repository.  For workflows, the default version is identified by the Git Reference (which could be a Git tag or a Git branch).  The default version can be set in the 'Versions' tab of a tool/workflow via radio buttons.
+
+Setting the default version affects several elements:
```

Comment:
Maybe "Setting the default version affects a number of elements including (but not limited to)"

We're bound to use this for more, but forget to update this. For example docker pull version